### ISSUE TITLE
ValueError invalid filename characters with wandb.log

### ISSUE THREAD

**Issue by bpetit2019** [wandb/wandb#4046]
I have a run with id=test_evaluation7.
When I use:
`wandb_run.log({difference ridgeline plot": fig})` it works correctly and log the plotly figure in media.

When I use:
`wandb_run.log({f"{title}-plot/difference ridgeline plot": fig})` I have this error message.

```
f"Media {key} is invalid. Please remove invalid filename characters"
ValueError: Media test_evaluation7-plot/difference ridgeline plot is invalid. Please remove invalid filename characters
```
Also, it works with this Image log:
`wandb_run.log({f"{suffix}/img_pred": wandb.Image(out_pred.get_image()[:, :, ::-1])})`

Why can't I use this format?

**Environment:**
Windows 10
wandb 0.12.7

**Comment by MBakirWB** [comment]
Hi @bpetit2019, I was unable to reproduce this error in wandb **0.12.7** or our latest **0.13.0**. I ran a quick test using a basic matplotlib figure and logged the figure using your f-string example: 

```
import wandb
import matplotlib.pyplot as plt
import numpy as np

t = np.arange(0.0, 2.0, 0.01)
s = 1 + np.sin(2 * np.pi * t)
fig, ax = plt.subplots()
ax.plot(t, s)

ax.set(xlabel='time (s)', ylabel='voltage (mV)',
       title='About as simple as it gets, folks')
ax.grid()

wandb.init(project="support")
title="test_evaluation7" #Referenced from error "test_evaluation7-plot/difference ridgeline plot is invalid"
wandb.log({f"{title}-plot/difference ridgeline plot": fig})
wandb.finish()
```

Can you please provide an example in a colab notebook I could use to replicate your issue. Thanks

**Comment by bpetit2019** [comment]
here is the kind of code I use:

```
import plotly.graph_objects as go
import wandb
import numpy as np

title="test_evaluation7"
run = wandb.init(project="wandb_project", resume="allow", allow_val_change=True, id=title)
fig = go.Figure()
data_line=[-3,-4,-9,4,10,3,5,-1]
fig.add_trace(
            go.Violin(
                x=np.array(data_line),
            )
)
run.log({f"{title}-plot/difference ridgeline plot": fig})
```
I think this should be enough to reproduce the error

**Comment by MBakirWB** [comment]
Hi @bpetit2019 , took a closer look. There was a regression present in wandb version 12.7 that disallowed use of "/" when logging from windows. This regression was fixed in wandb version 12.11, see [here](https://github.com/wandb/wandb/pull/3146). Please upgrade to latest wandb version.

**Comment by bpetit2019** [comment]
Thanks, I'll do that.
